creator-node/src/services/stateMachineManager/stateReconciliation/issueSyncRequest.jobProcessor.js

```diff
@@ -147,7 +166,8 @@ module.exports = async function ({ logger, syncType, syncRequestParameters }) {
       userWallet,
       secondaryEndpoint,
       primaryEndpoint: thisContentNodeEndpoint,
-      syncType
+      syncType,
+      syncMode: SYNC_MODES.SyncSecondaryFromPrimary
```

Comment:
is my understanding here correct? we issue a `SyncSecondaryFromPrimary` sync at the end regardless of the sync mode because it would either be 1 of 2 conditions:
1. `syncMode === SyncSecondaryFromPrimary`: that's what we'd want anyway
2. `syncMode === MergePrimaryAndSecondary`: we already did some logic earlier in the sync to sync secondary->primary and wipe secondary, so now we need to issue a `SyncSecondaryFromPrimary` sync